### PR TITLE
Add storybook API updates for several Platform components.

### PR DESCRIPTION
**Pull request by wilkie**
Some Platform team Storybook updates for existing components to the new Storybook API.

## Links

- jira ticket: [P20-40](https://codedotorg.atlassian.net/browse/P20-40)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/lib/ui/SchoolInfoConfirmationDialog.story.jsx [hunk 1]
```diff
@@ -2,6 +2,34 @@ import React from 'react';
...
+    formUrl: '',
+    authTokenName: 'auth_token',
+    authTokenValue: 'fake_auth_token',
+    existingSchoolInfo: {}
+  },
+  onClose: action('onClose callback'),
+  isOpen: true
+};
+
+/*
```
Comment (daynew) [inline apps/src/lib/ui/SchoolInfoConfirmationDialog.story.jsx]: Shouldn't this commented code be deleted?

File: apps/src/lib/ui/SchoolInfoInterstitial.story.jsx [hunk 2]
```diff
@@ -2,15 +2,46 @@ import React from 'react';
...
+  scriptData: {
+    formUrl: '',
+    authTokenName: 'auth_token',
+    authTokenValue: 'fake_auth_token',
+    existingSchoolInfo: {}
+  },
+  onClose: action('onClose callback')
+};
+
+/*
```
Comment (daynew) [inline apps/src/lib/ui/SchoolInfoInterstitial.story.jsx]: Commented code should be deleted

**Review by daynew — APPROVED** [review]
Looks good, just delete the commented code.

File: apps/src/lib/ui/SchoolInfoInterstitial.story.jsx (hunk 2 above)
Comment (wilkie) [reply apps/src/lib/ui/SchoolInfoInterstitial.story.jsx]: yes it should! thank you

**Pull request merged**